server/playerAPIsUtils.go

```diff
@@ -22,6 +22,10 @@ func getPlayerAPIs(db persistence.DB, players []model.Player) (map[model.PlayerI
 			if err != nil {
 				return nil, err
 			}
+			// TODO assign HandleAction callback here if the character is an NPC
+			if pAPI, ok := pAPI.(*interaction.NPCPlayer); ok {
+				pAPI.HandleActionCallback = HandleAction
```

Comment:
Instead, I was thinking we'd look at the `mode` of interacting. If it's set to `autorespond` or `npcmode` or something, then we'd set the `Info` as the func `HandleAction` so that `interaction.FromPlayerMeans` can do the lifting of assigning the callback for us